assets/src/edit-story/utils/setMinMax.js

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const setMinMax = (value, { MIN, MAX }) => Math.min(Math.max(value, MIN), MAX);
```

Comment:
The naming of this util is a bit confusing, perhaps `applyMinMax` would be more accurate.

Reply:
`[Math.]clamp` is an often used name for such a function.

Reply:
We already [have](https://github.com/google/web-stories-wp/search?q=clamp&unscoped_q=clamp) `clamp` with tests. Should we import it from Dashboard or copy to Editor (what's the general rule)?

Reply:
Copy it over, please

Reply:
Ok, NVM, just renamed it, they use array vs object argument - don't want to change it all now (maybe later if we decide to unify them).